python/ops.py

```diff
@@ -13,10 +13,34 @@
 # limitations under the License.
 """FFCC TensorFlow ops."""
 import math
-import sys
-
+import itertools
 import numpy as np
-import tensorflow.compat.v1 as tf
+import tensorflow as tf
+
+
+EPS = tf.constant(1e-9, dtype=tf.float32)
+
+
+def edge_kernel():
+  """Construct a set of 3x3 kernels for edge computation.
+  Convolving with these kernels produces the average of
```

Comment:
Add a newline here, and a newline before """ below. Can you also reformat text up to the 80 chars line length limit as well? You should be able to set up your code editor to enforce that.

https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
Fixed